packages/shared/src/components/Sidebar.tsx

```diff
@@ -106,16 +114,36 @@ const NavItem = classed(
 
 export default function Sidebar(): ReactElement {
   const { openSidebar, toggleOpenSidebar } = useContext(SettingsContext);
+  const { feedSettings } = useFeedSettings();
+  const { alerts, updateAlerts } = useContext(AlertContext);
   const [showSettings, setShowSettings] = useState(false);
+  const [isFilterOpen, setIsFilterOpen] = useState(false);
+
+  const showFilters = () => {
+    if (
+      alerts?.filter &&
+      (feedSettings?.includeTags?.length ||
+        feedSettings?.blockedTags?.length ||
+        feedSettings?.excludeSources?.length ||
+        feedSettings?.advancedSettings?.length)
+    ) {
+      updateAlerts({ filter: false });
+    }
+
+    setIsFilterOpen(true);
+  };
 
   const topMenuItems: SidebarMenuItems[] = [
     {
       key: 'Discover',
       items: [
         {
           icon: <ListIcon Icon={FilterIcon} />,
+          notif: alerts.filter && (
+            <BlueDot style={{ top: `-${0.125}rem`, right: `-${0.125}rem` }} />
```

Comment:
Tailwind has classes for these values:

```jsx
-top-0.5
-right-0.5
```

Reply:
Hmm. Let me try that again, I used it before it wasn't making any effect and had to do it this way.

Reply:
Oh might be because the alert has a default top offset already.